doc/glossary.rst

```diff
@@ -41,6 +41,12 @@ If you wish to add a missing term, please `create a new issue`_ or
         `Decoding`_ consists in predicting, from brain images, the conditions
         associated to trial.
 
+    Dictionary learning
+        `Dictionary learning`_ (or sparse coding) is a representation learning
+        method aiming at finding a sparse representation of the input data as
+        a linear combination of basic element called atoms. These atoms
+        composed the dictionary. Atoms are not required to be orthogonal.
```

Comment:
The identification of the atoms relies on a sparsity principle: maximally sparse representations of the dataset are sought for.

Reply:
Thanks!